modules/custom/az_news/config/install/views.view.az_news.yml

```diff
@@ -4,6 +4,7 @@ dependencies:
   config:
```

Comment:
is this file supposed to be part of this PR? modules/custom/az_news/config/install/views.view.az_news.yml

Reply:
yeah... no. been trying to figure out how to fix that